src/OpenTelemetry.Exporter.ZPages/ZPagesExporterHelperExtensions.cs

```diff
@@ -45,7 +45,7 @@ public static class ZPagesExporterHelperExtensions
             var zpagesExporter = new ZPagesExporter(exporterOptions);
 
             // TODO: Pick Simple vs Batching based on ZipkinExporterOptions
-            return builder.AddProcessor(new SimpleActivityProcessor(zpagesExporter));
+            return builder.AddProcessor(new BatchExportActivityProcessor(zpagesExporter));
```

Comment:
I guess ZPages can use the simple processor?

Reply:
i think so. I will close this pr then.

Reply:
If you look below, it seems to have its own processor?

Reply:
IMHO ZPages is not an exporter at all, it just a processor... 😆

Reply:
This is honestly the first time I have ever looked at the ZPages code but I think you are correct. This seems like a bug:

`return builder.AddProcessor(new SimpleActivityProcessor(zpagesExporter));`

Should be as you have above I think. Just 1 call to AddZPagesExporter which does AddProcessor(new ZPagesProcessor(new ZPagesExporter))?

Now there are probably also some bigger design issues at play here. Should all of this logic really be split across two things? Maybe it should all be in the processor or all in the exporter? Not sure 😄

Reply:
i will push this change doing this refactor, but maybe should be a good idea to check this "exporter"